k8s/crawler-deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: kompose convert
+    kompose.version: 1.17.0 (HEAD)
+  creationTimestamp: null
+  labels:
+    io.kompose.service: crawler
+  name: crawler
+spec:
+  replicas: 1
+  selector:
+    matchLabels: 
+      io.kompose.service: crawler
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        io.kompose.service: crawler
+    spec:
+      containers:
+      - args:
+        - monocle
+        - --elastic-conn
+        - elastic:9200
+        - crawler
+        - --config
+        - /etc/monocle/config.yaml
+        env:
+        - name: APP_ID
+          value: ${GITHUB_APP_ID:-}
+        - name: APP_KEY_PATH
+          value: ${GITHUB_APP_KEY_PATH:-/etc/monocle/app_key.rsa}
+        image: changemetrics/monocle_backend:latest
+        name: crawler
+        resources: {}
+        volumeMounts:
```

Comment:
The crawler needs the config and a RW volume to store dumps of Change that failed to be parsed.

Reply:
@guy9050 maybe this makes the PVC data be not only a config folder, but general days aswell

Reply:
@morucci there's a volume for config named "config-data" and "crawler-dump-data". Are we missing anything?